osler/assets/inventory/drug-list/DrugListTable.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import TableManager from '../../core/all-patients/TableManager';
+import CSRFToken from '../../util/CSRFToken';
+import compare from '../../util/compare';
+import simpleComparator from '../../util/simpleComparator';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Form from 'react-bootstrap/Form';
+import globalFilter from './globalFilter';
+import { DateTime } from 'luxon';
+
+
+function DrugListTable(props) {
+
+  const [state, setState] = useState({ 'show': false, 'drug': {} });
+
+  const handleClose = () => {
+    setState({ 'show': false, 'drug': {} });
+  }
+  const handleShow = (e, row) => {
+    setState({ 'show': true, 'drug': row });
+  }
+
+  const nameComparator = simpleComparator('name');
+  const categoryComparator = simpleComparator('category');
+  const expirationDateComparator = simpleComparator('expiration_date');
+  const stockComparator = simpleComparator('stock');
+  const doseComparator = React.useMemo(() => (rowA,rowB,columnId,desc) => {
```

Comment:
When I checked with the drugExample data, sorting is not working for stock, expiration date, dose. Sort by manufacturer should ignore case differences